jwst/stpipe/pipeline.py

```diff
@@ -206,6 +206,8 @@ def _precache_references_impl(self, model):
 
         fetch_types = sorted(set(self.reference_file_types) - set(ovr_refs.keys()))
 
+        self.log.info("Prefetching reference files for dataset: " + repr(model.meta.filename) + 
```

Comment:
At this point in the calling sequence can `model` ever be a `ModelContainer`, as opposed to a single data model? If so, then I don't think `model.meta.filename` is defined or at most might be the name of an ASN file. Or are `ModelContainer`s broken down into individual models before this is called?